kubernetes-itests/src/test/java/io/fabric8/kubernetes/PodIT.java

```diff
@@ -84,10 +82,11 @@
 
   private String currentNamespace;
 
-  private static final Logger logger = LoggerFactory.getLogger(PodIT.class);
+  private static Logger logger = Logger.getLogger(PodIT.class.getName());
```

Comment:
```suggestion
  private static final Logger logger = Logger.getLogger(PodIT.class.getName());
```

Reply:
If you changed this because of the Sonar warning, it was complaining about the order of the keywords.